client/blocks/reader-post-options-menu/blog-stickers.jsx

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { map, includes } from 'lodash';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { getBlogStickers } from 'state/selectors';
+import QueryBlogStickers from 'components/data/query-blog-stickers';
+import ReaderPostOptionsMenuBlogStickerMenuItem from './blog-sticker-menu-item';
+
+class ReaderPostOptionsMenuBlogStickers extends React.Component {
+	static propTypes = {
+		blogId: React.PropTypes.number.isRequired,
+	};
+
+	render() {
+		const blogStickersOffered = [ 'dont-recommend', 'broken-in-reader' ];
+		const { blogId, stickers } = this.props;
+
+		return (
+			<div className="reader-post-options-menu__blog-stickers">
+				{ map( blogStickersOffered, blogStickerName => (
+					<ReaderPostOptionsMenuBlogStickerMenuItem
+						key={ blogStickerName }
+						blogId={ blogId }
+						blogStickerName={ blogStickerName }
+						hasSticker={ includes( stickers, blogStickerName ) }
+					>
+						{ blogStickerName }
+					</ReaderPostOptionsMenuBlogStickerMenuItem>
+				) ) }
+				{ ! stickers && <QueryBlogStickers blogId={ blogId } /> }
+			</div>
+		);
+	}
+}
+
+export default connect( ( state, ownProps ) => {
+	return {
+		stickers: ownProps.blogId && ownProps.blogId > 0
```

Comment:
I usually find a ternary returning undefined smelly so thank you for the explanation!
 
> Just trying to be defensive not handing a null blogId to getBlogSelectors

thats one of the awesome things about lodash! Almost all of the functions can handle any bad input you give them and return something sensible.   You probably wouldn't need to put in explicit handling and the `get` function would just take care of it.

that said, this is all the smallest nit so 🤷‍♂️